packages/cdk8s/lib/app.ts

```diff
@@ -28,6 +32,39 @@ export class App extends Construct {
     this.outdir = options.outdir ?? process.env.CDK8S_OUTDIR ?? 'dist';
   }
 
+  protected onPrepare() {
```

Comment:
Let's merge these to into `synth` and avoid using synthesis altogther?

Reply:
Done